tasks/testing.rake

```diff
@@ -214,6 +214,15 @@ module MinitestSuite
   end
 end
 
+def execute_cmd_with_env(env, cmd)
+  if RUBY_VERSION.to_f > 2
```

Comment:
@Mogztter LOL sorry! 

I meant the first one, using `Kernel#system` directly. Or better removing the ruby-version check and raising an error in case it fails (which is the default behavior of `FileUtils#sh` that was used originally).